lib/gilmour/responder.rb

```diff
@@ -157,7 +158,7 @@ def emit_error(extra={})
       # Publish all errors on gilmour.error
       # This may or may not have a listener based on the configuration
       # supplied at setup.
-      if @backend.broadcast_errors
+      if @backend.broadcast_errors != false
```

Comment:
The backend can have a virtual method report_errors? which is overriden in the redis backend.